examples/simple-project/src/services/docs/__checks__/docs-search.spec.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
I think that the compiled TS files were accidentally included in the PR

Reply:
fixed